modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/util/fieldSupport.es.js

```diff
@@ -148,6 +148,18 @@ export const getFieldProperties = (
 					properties[fieldName] = value[editingLanguageId];
 				}
 			}
+			else if (type == 'validation') {
+				if (!value.errorMessage[editingLanguageId]) {
+					value.errorMessage[editingLanguageId] =
+						value.errorMessage[defaultLanguageId];
+				}
```

Comment:
@linolaoj, please add a line break after this line and feel free to forward this pull running the command `ci:forward`.

Thanks.